lib/api/_loaders/element-global.js

```diff
@@ -34,7 +34,8 @@ class ElementGlobal {
 
   static element({locator, testSuite, client, options}) {
     const instance = new ElementGlobal({testSuite, client, options});
-    instance.setLocator(locator);
+    const byInstance=Locator.create(locator);
```

Comment:
@yashPratp983 This is too early to use `Locator.create()` because if we use it here, we'll lose all the properties from `locator` like `index`, `timeout`, etc. which are being checked [here](https://github.com/nightwatchjs/nightwatch/blob/e1bed6afe4855800ba2921abb6bde6c8fe576fc4/lib/api/_loaders/element-global.js#L106) and set accordingly.

We should instead do this just above the line I linked in the issue, and only if `locator` is an instance of `Element`.